docs/content/Examples-Tutorials-Recipes/Pagination.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Pagination
+permalink: /recipes/pagination
+category: Examples & Tutorials
+subCategory: Queries
+menuOrder: 2
+---
+
+## Use case
```

Comment:
Maybe having an example of the data schema would be helpful?

Reply:
I have added the data schema to the `Data schema` section